### ISSUE TITLE
[ldc] Unable to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#1782]
```
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/ldc2.dir/build.make:62: bin/ldc2] Error 1
make[2]: Leaving directory '/media/tmp/crew/ldc-1.7.0-src.tar.gz.dir/ldc-1.7.0-src/build'
make[1]: *** [CMakeFiles/Makefile2:105: CMakeFiles/ldc2.dir/all] Error 2
make[1]: Leaving directory '/media/tmp/crew/ldc-1.7.0-src.tar.gz.dir/ldc-1.7.0-src/build'
make: *** [Makefile:163: all] Error 2
make: Leaving directory '/media/tmp/crew/ldc-1.7.0-src.tar.gz.dir/ldc-1.7.0-src/build'
ldc failed to build: `make V=0 -C build -j1` exited with 2
```

**Comment by lovedheart** [comment]
No problem on armv7l.

crew files ldc shows:
...
/usr/local/lib/libdruntime-ldc-shared.so.77
/usr/local/lib/libldc-jit-rt.a
/usr/local/lib/libldc-jit.so
/usr/local/lib/libldc-jit.so.2.0.77
/usr/local/lib/libldc-jit.so.77
/usr/local/lib/libldc-profile-rt.a
/usr/local/lib/libphobos2-ldc.a
/usr/local/lib/libphobos2-ldc-debug.a
/usr/local/lib/libphobos2-ldc-debug-shared.so
/usr/local/lib/libphobos2-ldc-debug-shared.so.2.0.77
/usr/local/lib/libphobos2-ldc-debug-shared.so.77
/usr/local/lib/libphobos2-ldc-shared.so
/usr/local/lib/libphobos2-ldc-shared.so.2.0.77
/usr/local/lib/libphobos2-ldc-shared.so.77
Total found: 586

**Comment by uberhacker** [comment]
Yes, I was able to get it working on armv7l also.  If you notice, there is no arm label on the issue.  Maybe there are missing dependencies?  Did you try on a fresh Chromebrew install yet?

**Comment by lovedheart** [comment]
I have tried a fresh crew install before compiling ldc.
